### DISCUSSION TITLE
Re-installing momentuHMM with R 4.0 and rtools40

### DISCUSSION THREAD

**Discussion by robbieemmet** [bmcclintock/momentuHMM#69] [Q&A] [answered]
Hi Brett,
I think when I tried to reinstall momentuHMM after updating R, RStudio, and Rtools, when I tried to install either from CRAN or GitHub, momentuHMM was trying to use the old version of Rtools, which I deleted from my computer. I imagine I can get around it by re-installing the old Rtools and using that, but I wanted to put this issue on your radar in case it wasn't yet. Thanks!
-Robbie Emmet

**Comment by robbieemmet** [comment]
OK, I did find that re-installing the old Rtools alongside the new one seemed to help for now.

**Comment by bmcclintock** [comment, marked as answer]
Unfortunately I'm not much help with Windows-specific issues. However, momentuHMM doesn't "know" you're using Rtools. Rather, it sounds like R isn't able to find the new compilers. Did you update your paths?

**Comment by robbieemmet** [comment]
Yes, I did update my paths, and fixed the issue!
